manifests/manage_app_pool.pp

```diff
@@ -249,7 +251,25 @@
         logoutput => true,
       }
     }
-
+    if ($apppool_recycle_fixed_requests) {
+      if (empty($apppool_recycle_fixed_requests)) {
```

Comment:
@AviIsraeli I'm not sure I understand the logic here: surely this should be just a number not an array? What's the usecase of setting it to an empty array?